Lib/fontTools/feaLib/builder.py

```diff
@@ -704,10 +704,7 @@ def set_language(self, location, language, include_default, required):
                 if key[:2] in self.get_default_language_systems_():
                     lookups = [l for l in lookups if l not in dflt_lookups]
                 self.features_.setdefault(key, []).extend(lookups)
-        if self.script_ == 'DFLT':
-            langsys = set(self.get_default_language_systems_())
-        else:
-            langsys = set()
+        langsys = set()
```

Comment:
This and the lines following can be simplified now: instead of creating a mutable empty set, calling add and then converting it to a frozenset, you could directly initialise the frozenset in one go